client/scripts/views/pages/new_snapshot_proposal/new_proposal_form.ts

```diff
@@ -142,10 +143,7 @@ const newThread = async (
     throw new Error(e.error);
   }
 
-  const wallet = await app.wallets.locateWallet(
-    author,
-    ChainBase.Ethereum
-  );
+  // const wallet = await app.wallets.locateWallet(author, ChainBase.Ethereum);
```

Comment:
can you explain this?

Reply:
The wallet variable wasn't being used anywhere, I commented it out. Do we need it for something?